### PR TITLE
add docs for resource preview

### PR DESCRIPTION
**Pull request by alexcodelf**
#197

### CODE REVIEW

File: docs/application/resource.md [hunk 1]
```diff
@@ -12,23 +12,27 @@ Resources belong to an environment, and you can view the list of resources in th
...
 
-![resource-list](/img/v0.5.0/application/resource/resource-list.png)
+![resource-list](/img/v0.6.0/application/resource/resource-list.png)
 
 ## View Resource Details
 
 1. Navigate to the `Resources` tab in the environment view.
 2. Find the resource you need to view and click its name to enter the detail page.
 
+On the resource details page, you can view basic information of the resource, the list of components, dependency graphs, etc. You can also view the latest run records or edit the resource.
```
Comment (gitlawr) [inline docs/application/resource.md:22]: On the resource details page, you can view the resource's basic information, component list, dependency graphs, and more.

File: docs/application/resource.md [hunk 2]
```diff
@@ -39,16 +43,32 @@ You can create a resource as draft.
...
+
+You can choose to save & preview resources when creating or editing resources. The resource preview can help you figure out the changes of the resource before do the real deployment.
+
+1. Navigate to the `Resources` tab in the environment view.
+2. Click `New Resource`.
+3. Select the resource type. If you want to create from template, enable the `Use Template` option, then select the template and version.
+4. Enter the `Name`, `Description`, and `Labels`. The name should be unique under the same environment.
+5. Fill in the configuration form, configuration items depends on the template or resource type you selected.
+6. Click the downward arrow to the right of the `Save & Deploy` button.
+7. Click `Save & Preview`.
```
Comment (gitlawr) [inline docs/application/resource.md:61]:
```
## Create Resource Preview

You can choose to save and preview resources when creating or editing them. The resource preview helps you understand the changes to the resource before performing the actual deployment.

1. Navigate to the `Resources` tab in the environment view.
2. Click on `New Resource`.
3. Select the resource type. If you wish to create a resource from a template, enable the `Use Template` option, then select the desired template and its version.
4. Enter the `Name`, `Description`, and `Labels`. Ensure the name is unique within the same environment.
5. Fill in the configuration form. The configuration items will depend on the selected template or resource type.
6. Click the downward arrow next to the `Save & Deploy` button.
7. Select `Save & Preview`.
```

File: docs/application/resource.md [hunk 3]
```diff
@@ -57,33 +77,35 @@ Resources can depend on each other by using output references. The referenced en
...
-![resource-rollback](/img/v0.5.0/application/resource/resource-rollback.png)
+![resource-rollback](/img/v0.6.0/application/resource/resource-rollback.png)
 
 4. Choose the historic version you need to roll back to.
 5. Confirm the configuration changes for the rollback operation, click the `Rollback` button to complete the rollback operation.
 
-![resource-rollback2](/img/v0.5.0/application/resource/resource-rollback2.png)
+![resource-rollback2](/img/v0.6.0/application/resource/resource-rollback2.png)
+
+> You can also rollback with preview before applying the changes.
```
Comment (gitlawr) [inline docs/application/resource.md:95]: You can also preview a rollback before applying the changes.

File: docs/application/resource.md [hunk 4]
```diff
@@ -170,6 +192,16 @@ View the component list in the resource detail page:
...
 For more details on dependency graphs, please refer to [Dependency Graph](/application/graph).
 
-![resource-component-graph](/img/v0.5.0/application/resource/resource-component-graph.png)
+![resource-component-graph](/img/v0.6.0/application/resource/resource-component-graph.png)
+
+![resource-component-graph2](/img/v0.6.0/application/resource/resource-component-graph2.png)
+
+## Resource Runs
+
+Every operation such as deployment, rollback, start, and stop of a resource will generate a run record. You can view all run records and their details on the resource details page.
```
Comment (gitlawr) [inline docs/application/resource.md:201]: Every operation, including deployment, rollback, starting, and stopping a resource, generates a run record.

File: docs/application/resource.md [hunk 5]
```diff
@@ -170,6 +192,16 @@ View the component list in the resource detail page:
...
+
+![resource-run-list](/img/v0.6.0/application/resource/resource-run-list.png)
+
+The resource run includes information such as the status of the run, start time, duration, type of operation, operator, component changes in resources, resource configuration, logs, change history, etc. You can view the detailed information of the run by clicking the operation button.
```
Comment (gitlawr) [inline docs/application/resource.md:205]: The resource run includes information such as the run's status, start time, duration, operation type, operator, component changes in resources, resource configuration, logs, change history, and more. You can view detailed information about the run by clicking the operation button.

File: docs/application/resource.md (hunk 4 above)
Comment (alexcodelf) [reply docs/application/resource.md:201]: updated.

File: docs/application/resource.md (hunk 3 above)
Comment (alexcodelf) [reply docs/application/resource.md:95]: updated.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**